Exercise_B/index.html

```diff
@@ -1,34 +1,62 @@
 
-    <!-- Link and CDN required for bootstrap -->
+<head>
+  <meta charset="utf-8" />
+  <title>Flower Shop BS</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
 
-    <!-- Google Font - Lato -->
-    <link
-      href="https://fonts.googleapis.com/css?family=Lato"
-      rel="stylesheet"
-    />
-    <!-- Custom Stylesheet -->
-    <link rel="stylesheet" href="./assets/style/stylesheet.css" />
-  </head>
+  <!-- Link and CDN required for bootstrap -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous" />
+  <!-- Google Font - Lato -->
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" />
+  <!-- Custom Stylesheet -->
+  <link rel="stylesheet" href="./assets/style/stylesheet.css" />
+</head>
 
-  <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
-
-    <!--Four containers with img / text-->
-    <!-- 4 containers -->
-
-    <!-- Payments Div -->
+<body>
+  <!--Header (flower shop logo block element)-->
+  <header>
+    <img src="/Exercise_B/assets/images/logo.png" alt="">
+  </header>
+  <!--Banner Jumbotron of the image-->
+  <section class="d-flex flex-column justify-content-center align-items-center text-white"
+    style="background-image: url(/Exercise_B/assets/images/flowershop.jpg); height: 70vh;">
+    <h1>Beautiful Flowers</h1>
+    <a class="btn btn-success" href="#" role="button">Learn More</a>
+  </section>
+  <!--Four containers with img / text-->
+  <!-- 4 containers -->
+  <div class="row">
+    <!-- Flower Div -->
+    <div class="col-sm-3">
+      <img src="/Exercise_B/assets/images/flower-icon.png" alt="">
+      <p>Flowers for every kind of situation</p>
+    </div>
+    <!-- Payment Div -->
+    <div class="col-sm-3">
+      <img src="/Exercise_B/assets/images/payment-icon.png" alt="">
+      <p>All major payments kinds accepted</p>
+    </div>
     <!-- Consultation Div -->
-
+    <div class="col-sm-3">
+      <img src="/Exercise_B/assets/images/phone-icon.png" alt="">
+      <p>Free Consultation</p>
+    </div>
     <!-- Delivery Div -->
+    <div class="col-sm-3">
+      <img src="/Exercise_B/assets/images/truck-icon.png" alt="">
+      <p>City wide delivery</p>
+    </div>
+  </div>
+
+
+  <!--Scripts required for bootstrap CDN-->
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous">
+  </script>
+</body>
 
-    <!--Scripts required for bootstrap CDN-->
-  </body>
-</html>
+</html>
```

Comment:
Great Job on having the items flex and be responsive, but do note that the images do not load as you have not added a ../ to the start of the href and also for the main banner you added background-image inside the html. This can be referred to but through a CSS File.